README.md

```diff
@@ -1,65 +1,71 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# PERSONAL PORTFOLIO
 
-> One paragraph statement about the project.
+> A "portfolio" HTML-5 & CSS3 repository built during the portal learning module. 
 
-![screenshot](./app_screenshot.png)
+![screenshot](./images/Nemwellandingpage1.png)
 
-Additional description about the project and its features.
+It is a show of my understanding of Gitflow, use of Linters, HTML5 semantics, CSS3 flexbox and figma.
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- HTML-5
+- CSS3
+- Linters
+- Figma
 
 ## Live Demo
 
-[Live Demo Link](https://livedemo.com)
+[Live Demo Link](https://nemwel-boniface.github.io/personal-portfolio/)
```

Comment:
- [ ] Your live demo link does not work.  Kindly correct it so the users have a way to try your work.

Reply:
Hello @Meltrust , Thank you for this review.
I had created the Github pages and set them to the main branch so that when I merge my pull requests the pages can be viewed there. I have made the changes now and it can now be viewed from the landingpage1 branch. I am now making the other changes
![PRCHANGE1](https://user-images.githubusercontent.com/86318284/148385633-c2969e5a-2143-47e4-bb29-b718da97c265.png)
changes.